docs/public/specifications/tests/DNSSEC-TP/dnssec01.md

```diff
@@ -191,14 +193,26 @@ queries follow, unless otherwise specified below, what is specified for
           the *Algo 2 DS* set.
        6. Else, add IP address and the key tag to the *Non-Algo 2 DS* set.
 
-6.  For each of the sets matching each of the following message tags do if the set
+6.  To be considered in the steps below:
+    * If when outputting a message tag a list of name servers ("ns_list" in the
+      [message tag specification][Summary]) is included, the name or names of the
+      name server should be looked up from the IP address in the
+      *Parent Name and IP* set (if multiple names, it results from multiple name
+      server names resolving into the same IP address).
+    * For message tags with "ns_list", name and IP address pairs should be
+      included, possibly multiple pairs for the same IP address.
```

Comment:
In previous (recent) specifications we took the habit of simply stating the sentence below in the `Summary` section, instead of over-complicating the test procedure with specificities such as these. E.g. see DNSSEC10 spec.

> The name server names are assumed to be available at the time when the msgid is created, if the argument name is "ns" or "ns_list" even when in the "[Test procedure]" below it is only referred to the IP address of the name servers.